config.json

```diff
@@ -684,6 +684,64 @@
                }
             }
          ]
+      },
+      {
+         "url":"https://github.com/ebuzzing/lib-front-models.git",
+         "rebase":[
+            {
+               "from":"master",
+               "to":"feat.*",
```

Comment:
I hesitated :) That looked to be a regex but I wanted to keep the same logic as the other configs.
But your right it's better in one rule! I'll made the changes :)